ui/src/app/views/workspace/component.ts

```diff
@@ -300,4 +341,18 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     return this.accessLevel === WorkspaceAccessLevel.OWNER;
   }
 
+  sendFilesToNotebookServer(notebooks): void {
+    const fileList: Array<FileDetail> = notebooks.filter((item) => item.push);
+    this.clusterService
+        .localizeNotebook(this.workspace.namespace, this.workspace.id, fileList)
+        .subscribe(() => {
+          this.alertCategory = 'alert-success';
+          this.alertMsg = 'File(s) have been saved';
+          this.showAlerts = true;
```

Comment:
I don't see anything that would clear `this.alert*` values. Should there be something? Fine if it's a TODO, if we expect this to change significantly.

Reply:
Is this addressed? e.g. even just a TODO or comment somewhere noting that this is temporary?

Replying on individual comments would really be useful here to help me understand what has been addressed vs places of disagreement. On my reviews I'd generally reply to every single comment for clarity, though that's not a hard requirement. As the sender of a PR, this also helps me to track which comments I have addressed so far (you can buffer them via "start review" then send them once you're done).

Reply:
Settimeout has been added to revert the value of alerts (category msg and showError) after 5 sec